internal/configs/virtualserver.go

```diff
@@ -2452,7 +2455,11 @@ func generateProxySSLName(svcName, ns string) string {
 	return fmt.Sprintf("%s.%s.svc", svcName, ns)
 }
 
-func isTLSEnabled(u conf_v1.Upstream, spiffeCerts bool) bool {
+func isTLSEnabled(u conf_v1.Upstream, spiffeCerts, spiffeEgress bool) bool {
```

Comment:
Could you add some comments around this function? It's not immediately clear to me what we're checking and why if `spiffeEgress` is set TLS is disabled and I think a description will help in the future 🙂

Reply:
Good catch, yeah that seems really confusing. Updated for clarity.